### PR TITLE
Task 1 part 2 implement design elements using html css

### PR DESCRIPTION
**Pull request by ashrafabuodeh**

### CODE REVIEW

File: task_1/fonts/google-sans-cufonfonts-webfont/example.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+<h1 style="font-family:'Product Sans Thin Italic';font-weight:normal;font-size:42px">AaBbCcDdEeFfGgHhŞşIıİi Example</h1>
+<h1 style="font-family:'Product Sans Light Italic';font-weight:normal;font-size:42px">AaBbCcDdEeFfGgHhŞşIıİi Example</h1>
+<h1 style="font-family:'Product Sans Medium Italic';font-weight:normal;font-size:42px">AaBbCcDdEeFfGgHhŞşIıİi Example</h1>
+<h1 style="font-family:'Product Sans Bold';font-weight:normal;font-size:42px">AaBbCcDdEeFfGgHhŞşIıİi Example</h1>
+<h1 style="font-family:'Product Sans Bold Italic';font-weight:normal;font-size:42px">AaBbCcDdEeFfGgHhŞşIıİi Example</h1>
+<h1 style="font-family:'Product Sans Black Italic';font-weight:normal;font-size:42px">AaBbCcDdEeFfGgHhŞşIıİi Example</h1>
+
+
+</body>
+</html>
```
Comment (abood454) [inline task_1/fonts/google-sans-cufonfonts-webfont/example.html]: add one space

File: task_1/index.html [hunk 2]
```diff
@@ -4,40 +4,150 @@
...
+                    <a class="Nav-items menue-item  " href="http://" onclick="onMenueItemClick('itemHome')" id="itemHome">Home</a>
+                    <a class="Nav-items menue-item " href="http://"  onclick="onMenueItemClick('itemAboutUs')" id="itemAboutUs">About Us</a>
+                    <a class="Nav-items menue-item " href="http://" onclick="onMenueItemClick('itemOurlands')" id="itemOurlands">Ourlands</a>
+                    <a class="Nav-items menue-item " href="http://" onclick="onMenueItemClick('itemSuppliers')" id="itemSuppliers">Suppliers</a>
+                    <a class="Nav-items menue-item " href="http://" onclick="onMenueItemClick('itemProducts')" id="itemProducts">Products</a>
+                    <a class="Nav-items menue-item " href="http://" onclick="onMenueItemClick('itemContactUs')" id="itemContactUs">Contact Us</a>
+            </div>
+                    <div>
+                    <img src="assets/shopping-cart.png" alt="shopping cart " class="same-size" >
+                    <img  src="assets/profile Image.png" alt="profileImages" class="img-profile" >
```
Comment (abood454) [inline task_1/index.html]: the width and height of the profile image need to change

File: task_1/index.html [hunk 3]
```diff
@@ -4,40 +4,150 @@
...
+            <pre class="Ps">1 Bag                                            $69.99</pre>
+            </div>
+            </div>
             <div class="hero-banner">
                 <div class="text-banner">
-                    <p>Be Active</p>
-                    <p>Barnsley Brew Coffee</p>
-                    <p>Coffee is a brewed drink prepared from roasted coffee beans, the seeds of berries from certain
+                    <p class="text-banner-color">Be Active</p>
+                    <p class="Ps fs">Barnsley Brew Coffee</p>
```
Comment (abood454) [inline task_1/index.html]: the names aren't informative.

File: task_1/style.css [hunk 4]
```diff
@@ -1,24 +1,82 @@
...
 }
 
 .hero-image{
-  border: 1px solid black;
+  background-color: white;
   min-width: 320px;
+  margin-left: 10px;
+}
+
+.img-profile{
```
Comment (araraltawarah98) [inline task_1/style.css:62]: Set same value in height and width

File: task_1/style.css [hunk 5]
```diff
@@ -1,24 +1,82 @@
...
+}
+
+.hero-text{
+  text-align: center;
+}
+
+.Gs{
+  font-family: 'google sans';
+  text-align: center  ;
+
```
Comment (araraltawarah98) [inline task_1/style.css]: please remove space

File: task_1/index.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+            if(document.getElementsByClassName('Nav-items')[i].classList.contains('active')){
+                document.getElementsByClassName('Nav-items')[i].classList.remove('active');
+         }
+        }
+
+        document.getElementById(elementId).classList.add('active');
+        
+       
+
+
```
Comment (araraltawarah98) [inline task_1/index.js]: remove spaces

File: task_1/style.css [hunk 7]
```diff
@@ -1,24 +1,82 @@
...
+}
+
+
+@font-face{
+  font-family: "products san" ;
+  src: url(./fonts/FontsFree-Net-ProductSans-Regular.ttf);
+}
 
 .wrapper{
+  background: #E5E5E5;
```
Comment (abood454) [inline task_1/style.css:13]: there is a white space in the top of the page , i think you should give the background to the body.

File: task_1/index.html [hunk 8]
```diff
@@ -4,40 +4,150 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>layout structuring</title>
+    <title>Coffee Style</title>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN" crossorigin="anonymous">
     <link rel="stylesheet" href="./style.css">
+
```
Comment (abood454) [inline task_1/index.html]: extra space.

File: task_1/index.html [hunk 9]
```diff
@@ -4,40 +4,150 @@
...
+                <div>
+                    <img src="assets/MOcha Late 02.png" alt="Mocha Late" srcset="">
+                    <p class="Ps">Mocha Late</p>
+                    <p class="Gs">Coffee is a brewed drink prepared 
+                        from roasted coffee beans, the seeds of berries from certain Coffea species.</p>
+                </div>
+            </div>
+            <div class="coffee-item">
+                <div>
+                    <img src="assets/Double espresso-1.png" alt="Double espresso" srcset="">
```
Comment (araraltawarah98) [inline task_1/index.html]: please remove any unnecessary attributes

**Review by abood454 — CHANGES_REQUESTED** [review]
the space between elements has to be redone and redesign some parts

File: task_1/style.css [hunk 10]
```diff
@@ -1,51 +1,109 @@
...
+}
+.Nav-items{
+  margin: 15px;
+  text-decoration: none;
+}
 .main{
-  border: 1px solid black;
   display: flex;
+  margin-top: 60px;
   padding-bottom: 32px;
```
Comment (Mohsen-Awad) [inline task_1/style.css]: use shorthand padding

File: task_1/style.css [hunk 11]
```diff
@@ -1,51 +1,109 @@
...
   width: 217px;
 }
-
 .grid-items{
-  border: 1px solid black;
+  background: linear-gradient(180deg, rgba(228, 239, 233, 0) 0%, #ECF8F1 6.25%);
   display: grid;
   grid-gap: 32px;
   grid-template-columns: repeat(auto-fill, minmax(270px, 1fr));
   padding-top: 3rem;
```
Comment (Mohsen-Awad) [inline task_1/style.css]: use shorthand padding

File: task_1/style.css [hunk 12]
```diff
@@ -1,51 +1,109 @@
...
+  height: 10px;
+  width: 90px;
+}
+.text-banner-color{
+ color: red;
+ font-family: "products san";
+}
+.title-Barsley-coffee{
+  font-size: 40px;
+  margin-top: 20px;
```
Comment (Mohsen-Awad) [inline task_1/style.css]: use shorthand margin

File: task_1/style.css [hunk 13]
```diff
@@ -1,51 +1,109 @@
...
+  text-align: start;
+}
+.coffeebens{
+  margin-left: 45px;
```
Comment (Mohsen-Awad) [inline task_1/style.css]: also use shorthand margin

**Review by Mohsen-Awad — CHANGES_REQUESTED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

File: task_1/fonts/google-sans-cufonfonts-webfont/style.css [hunk 14]
```diff
@@ -0,0 +1,96 @@
...
+    src: local('Product Sans Bold Italic'), url('ProductSans-BoldItalic.woff') format('woff');
+    }
+    
+
+    @font-face {
+    font-family: 'Product Sans Black Italic';
+    font-style: normal;
+    font-weight: normal;
+    src: local('Product Sans Black Italic'), url('ProductSans-BlackItalic.woff') format('woff');
+    }
```
Comment (araraltawarah98) [inline task_1/fonts/google-sans-cufonfonts-webfont/style.css]: add end of file line

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/fonts/google-sans-cufonfonts-webfont/style.css [hunk 15]
```diff
@@ -0,0 +1,96 @@
+/* #### Generated By: http://www.cufonfonts.com #### */
+
+    @font-face {
```
Comment (laithyounes-archive) [inline task_1/fonts/google-sans-cufonfonts-webfont/style.css]: this file is not formatted

**Review by Mohsen-Awad — APPROVED** [review]

**Review by abood454 — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

File: task_1/index.js [hunk 16]
```diff
@@ -0,0 +1,12 @@
...
+    const NumberOfItem = document.getElementsByClassName('Nav-items').length;
+         for(let i = 0 ; i < NumberOfItem; i++){
+            if(document.getElementsByClassName('Nav-items')[i].classList.contains('active')){
+                document.getElementsByClassName('Nav-items')[i].classList.remove('active');
+         }
+        }
+        document.getElementById(elementId).classList.add('active');       
+    } 
+    
+
```
Comment (mmasaid) [inline task_1/index.js:10]:
```suggestion
```

File: task_1/index.js [hunk 17]
```diff
@@ -0,0 +1,12 @@
+const onMenueItemClick = (elementId)=>{
+     const itemClick = document.getElementById(elementId) ;
+    const NumberOfItem = document.getElementsByClassName('Nav-items').length;
```
Comment (mmasaid) [inline task_1/index.js]:
```suggestion
    const navItems = document.getElementsByClassName('Nav-items').length;
```

File: task_1/style.css [hunk 18]
```diff
@@ -1,51 +1,141 @@
...
+
+.menueItem:hover{
+  background-color: red;
+}
+
+.active{
+  background-color: yellow;
+}
+
+.navItems{
```
Comment (mmasaid) [inline task_1/style.css]: for the class, ids, image names use kabab case and be consistent 
**please update the names for class and ids** 
```suggestion
.nav-items{
```

**Review by mmasaid — CHANGES_REQUESTED** [review]
LGTM just need to be consistent with names conventions 
let's use Kabab case for the classes and ids and the image names

**Review by mmasaid — APPROVED** [review]
LGTM

**Pull request merged**